package.json

```diff
@@ -114,7 +114,7 @@
     "rollup-plugin-commonjs": "^10.0.2",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-terser": "^5.1.1",
-    "rollup-plugin-typescript2": "^0.25.1",
+    "rollup-plugin-typescript2": "^0.25.2",
```

Comment:
@P0lip Looks like version 0.25.1 disappeared from npm registry...

![image](https://user-images.githubusercontent.com/92363/68297634-c3027000-0097-11ea-915d-54953802c474.png)


...weird...

Reply:
Interesting. Looks like author marked it as deprecated.
https://github.com/ezolenko/rollup-plugin-typescript2/releases
Let's update to 0.25.2 then.